Dockerfile

```diff
@@ -1,18 +1,34 @@
 FROM sailfishos-platform-sdk-base
 MAINTAINER Lucien Xu <sfietkonstantin@free.fr>
 
-# Add nemo in sudoers without password
-RUN chmod +w /etc/sudoers && echo "nemo ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers && chmod -w /etc/sudoers
+ARG extra_packages
```

Comment:
I'm not sure about the need of this. Tar is needed to untar the sdk images, so more like a build dependency requirement.

Reply:
As option one can include openssh-server here